Sources/PeripheryKit/Frontend/Commands/ScanCommand.swift

```diff
@@ -221,6 +223,10 @@ public struct ScanOptions: OptionsProtocol {
             <*> mode <| Option(key: "disable-update-check",
                                defaultValue: BoolValue(!config.updateCheck),
                                usage: "Disable checking for updates")
+
+            <*> mode <| Option(key: "fail-on-warnings",
+                               defaultValue: BoolValue(config.failOnWarnings),
+                               usage: "Make sure command fails if any warnings are encountered")
```

Comment:
Change this to "Exit with non-zero status if any unused code is found"

Reply:
Done.